src/components/organisms/pageTemplate.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react";
+import NavBar from "./NavBar";
+import Footer from "./Footer";
+
+const PageTemplate = ({ children }: { children: ReactNode }) => {
```

Comment:
stict to the "function" key word when defining functional components.
```suggestion
function PageTemplate({ children }: { children: ReactNode } ) {
```